future-apps/payroll/.gitignore

```diff
@@ -0,0 +1,7 @@
+.cache
```

Comment:
Hmm, this should probably be in the payroll app's PR :).

Reply:
Agreed. I noticed that after I pushed 😅